appengine-standard-java8/kotlin-appengine-standard/pom.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+  <packaging>war</packaging>
+  <version>1.0-SNAPSHOT</version>
+
+  <groupId>com.google.appengine.demos</groupId>
+  <artifactId>kotlin-appengine-Standard</artifactId>
+
+  <dependencies>
```

Comment:
You might wish to grab the `<parent>` from another sample so that all the samples can be tested. (also add it to head).  (we usually comment that the section is optional.)